src/main/java/step2/type/WinningType.java

```diff
@@ -0,0 +1,42 @@
+package step2.type;
+
+import java.util.stream.Stream;
+
+public enum WinningType {
+    RANK_ONE(6, 2000000000),
```

Comment:
```suggestion
    RANK_ONE(6, 2_000_000_000),
```
위와 같이 표기하면 가독성을 향상시킬 수 있겠죠? 😊